src/templates/finops-hub/modules/dataFactory.bicep

```diff
@@ -4593,6 +4596,7 @@ resource pipeline_ExecuteIngestionETL 'Microsoft.DataFactory/factories/pipelines
         ]
         userProperties: []
         typeProperties: {
+          batchCount: dataExplorerIngestionCapacity // Concurrency limit
```

Comment:
@MSBrett I would rather query for the capacity than use our own SKU logic. That would continue to work if ADX changes their capacity per SKU.